frontend/plan/package.json

```diff
@@ -15,7 +15,11 @@
   "dependencies": {
     "@material-ui/core": "^4.5.2",
     "@sentry/browser": "^5.19.2",
+    "@types/react": "^16.9.49",
+    "@types/react-dom": "^16.9.8",
     "@types/styled-components": "^5.1.3",
+    "add": "^2.0.6",
```

Comment:
were these (`add`, `and`) on purpose? Also don't know if `yarn` needs to be added either